tasks/register.yml

```
@@ -31,6 +31,15 @@
     --docker-privileged="{{ gitlab_runner.docker_privileged | default(false) }}"
     --docker-tlsverify="{{ gitlab_runner.docker_tls_verify | default(false) }}"
     --docker-shm-size="{{ gitlab_runner.docker_shm_size | default(0) }}"
+    {% if gitlab-runner.docker_cpus is defined %}
```

Need to be an underscore:

```suggestion
    {% if gitlab_runner.docker_cpus is defined %}
```